logs/storage/MainchainGatewayV3.sol:MainchainGatewayV3.log

```diff
@@ -14,9 +14,9 @@ src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:dailyWithdrawalLimit (st
 src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:lastSyncedWithdrawal (storage_slot: 61) (offset: 0) (type: mapping(address => uint256)) (numberOfBytes: 32)
 src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:lastDateSynced (storage_slot: 62) (offset: 0) (type: mapping(address => uint256)) (numberOfBytes: 32)
 src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:______gap (storage_slot: 63) (offset: 0) (type: uint256[50]) (numberOfBytes: 1600)
-src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_initialized (storage_slot: 113) (offset: 0) (type: uint8) (numberOfBytes: 1)
-src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_initializing (storage_slot: 113) (offset: 1) (type: bool) (numberOfBytes: 1)
-src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_roles (storage_slot: 114) (offset: 0) (type: mapping(bytes32 => struct AccessControl.RoleData)) (numberOfBytes: 32)
+src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_roles (storage_slot: 113) (offset: 0) (type: mapping(bytes32 => struct AccessControl.RoleData)) (numberOfBytes: 32)
+src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_initialized (storage_slot: 114) (offset: 0) (type: uint8) (numberOfBytes: 1)
+src/mainchain/MainchainGatewayV3.sol:MainchainGatewayV3:_initializing (storage_slot: 114) (offset: 1) (type: bool) (numberOfBytes: 1)
```

Comment:
wrong storage slot

Reply:
solved by removing `Initializable` extension. However, we might have to investigate why as this corrupt current storage slot.